inventory/host_vars/openfoodnetwork.org.au/config.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
 domain: openfoodnetwork.org.au
-host_id: au-prod
+host_id: au-prod4
```

Comment:
Why do we need to update this ?

Reply:
I added the number to during switchover, which was really helpful to be sure which server was which.

It's still slightly relevant now, because we still have two vps in the hosting console:
![Screenshot 2024-12-16 at 10 38 13 AM](https://github.com/user-attachments/assets/ee4f89a6-a7e0-46f1-8d15-6616731398bf)

But prod3 should be shutdown soon.. so yeah let's not have a superfluous number here. I will remove.